### PR TITLE
updated to net461

### PR DESCRIPTION
**Pull request by acesnik**

### CODE REVIEW

File: FlashLFQ/FlashLFQ.csproj [hunk 1]
```diff
@@ -5,10 +5,14 @@
   </PropertyGroup>
   
   <PropertyGroup Condition="'$(TargetFramework)' == 'net461'">
-    <DefineConstants>NETFRAMEWORK</DefineConstants>
+    <DefineConstants>DEBUG;NETFRAMEWORK;NET461</DefineConstants>
```
Comment (rmillikin) [inline FlashLFQ/FlashLFQ.csproj]: don't think this needs to be changed

File: FlashLFQ/FlashLFQ.csproj [hunk 2]
```diff
@@ -5,10 +5,14 @@
...
     <PlatformTarget>x64</PlatformTarget>
   </PropertyGroup>
   
+  <PropertyGroup Condition="'$(Configuration)|$(TargetFramework)|$(Platform)'=='Debug|net461|AnyCPU'">
+    <OutputPath>bin\Debug\net461\</OutputPath>
+  </PropertyGroup>
+
```
Comment (rmillikin) [inline FlashLFQ/FlashLFQ.csproj]: I think this section is not needed

File: Test/packages.config [hunk 3]
```diff
@@ -1,5 +1,5 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <packages>
-  <package id="NUnit" version="3.9.0" targetFramework="net45" />
-  <package id="NUnit3TestAdapter" version="3.9.0" targetFramework="net45" />
+  <package id="NUnit" version="3.10.1" targetFramework="net471" />
```
Comment (rmillikin) [inline Test/packages.config]: should be 461? do you need to get whatever NUnit nuget package uses 461?

File: TestThermo/packages.config [hunk 4]
```diff
@@ -1,6 +1,6 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <packages>
-  <package id="NUnit" version="3.9.0" targetFramework="net45" />
-  <package id="NUnit3TestAdapter" version="3.9.0" targetFramework="net45" />
-  <package id="System.ValueTuple" version="4.4.0" targetFramework="net45" />
+  <package id="NUnit" version="3.10.1" targetFramework="net461" />
+  <package id="NUnit3TestAdapter" version="3.10.0" targetFramework="net461" />
+  <package id="System.ValueTuple" version="4.4.0" targetFramework="net45" requireReinstallation="true" />
```
Comment (rmillikin) [inline TestThermo/packages.config]: does it matter if this is targeting net45?

File: mzLib.sln [hunk 5]
```diff
@@ -61,11 +61,13 @@ Global
 		{D7C69058-1FE4-49BD-B994-B08C1DFB21A4}.Debug|Any CPU.ActiveCfg = Debug|x64
 		{D7C69058-1FE4-49BD-B994-B08C1DFB21A4}.Debug|Any CPU.Build.0 = Debug|x64
 		{D7C69058-1FE4-49BD-B994-B08C1DFB21A4}.Release|Any CPU.ActiveCfg = Release|x64
+		{D7C69058-1FE4-49BD-B994-B08C1DFB21A4}.Release|Any CPU.Build.0 = Release|x64
```
Comment (rmillikin) [inline mzLib.sln]: I think all the changes in this file should be undone

Comment (acesnik) [reply mzLib.sln]: The build order wasn't working for me in "Release" like it was in "TestAndRelease" and "Debug" -- some things just weren't getting built. These changes come from checking all of the projects in editing the "Release" Build Order.

File: FlashLFQ/FlashLFQ.csproj (hunk 1 above)
Comment (acesnik) [reply FlashLFQ/FlashLFQ.csproj]: You're probably right. This was automatically generated.

File: Test/packages.config (hunk 3 above)
Comment (acesnik) [reply Test/packages.config]: Hmm, good thoughts. I don't know. The tests do run with this version.

File: TestThermo/packages.config (hunk 4 above)
Comment (acesnik) [reply TestThermo/packages.config]: Good question. I'm not sure there's another option for this one. It does build...

**Pull request merged**